rules-tests/Php80/Rector/Identical/StrEndsWithRector/Fixture/substr_not_prefix.php.inc

```diff
@@ -6,11 +6,12 @@ class SubstrNotPrefix
 {
     public function run()
     {
-        $isMatch = $needle !== substr($haystack, -strlen($needle));
+        $isNotMatch = substr($haystack, -strlen($needle)) !== $needle;
```

Comment:
I think you can add another line for flip check like other fixtures, so no need another file:

```suggestion
        $isNotMatch = substr($haystack, -strlen($needle)) !== $needle;
        $isNotMatch = $needle !== substr($haystack, -strlen($needle));
```

Reply:
The positive match on substr is using two separate files at the moment - `fixture.php.inc` and `substr_other_way.php.inc`. Should I move this to a single file at the same time?

Reply:
I think yes, for grouping case as both equals, just flipped compare. `substr_other_way.php.inc` content can be added to `fixture.php.inc`, then remove `substr_other_way.php.inc`.